misc/vim/plugin/nit.vim

```diff
@@ -310,6 +334,8 @@ fun Nitdoc()
 			silent put = ''
 		endif
 	endfor
+	execute 0
```

Comment:
It's actually doing a `:0` to go to the first line. It's `execute delete` the real danger!